package.json

```diff
@@ -61,5 +61,8 @@
 	"resolutions": {
 		"minimist": "1.2.6",
 		"adm-zip": "0.5.9"
+	},
+	"dependencies": {
+		"textarea-caret": "^3.1.0"
```

Comment:
this dependency in this package is unnecessary

Reply:
I updated this branch and these files came out of nowhere, i have now removed them you can check please